core/src/builtin_widgets/key.rs

```diff
@@ -71,10 +72,11 @@ pub struct KeyWidget<V = ()> {
   #[declare(convert=into)]
   pub key: Key,
   pub value: Option<V>,
+
   #[declare(default)]
   before_value: Option<V>,
   #[declare(default)]
-  status: KeyStatus,
+  status: RefCell<KeyStatus>,
```

Comment:
Should not wrap with `RefCell`, status is a public status, others may depend on it across methods like `is_enter`.